app/Http/Controllers/Api/FacultyController.php

```diff
@@ -38,6 +38,14 @@ public function create(Request $request)
         return response()->json(['error' => 'This faculty already exist'], 409);
     }
 
+    public function check()
```

Comment:
check? why not getFacultiesWithDepartments?

Reply:
because am checking, not getting. Hower it being well described on api routes page